dev/com.ibm.ws.security.fat.common/src/com/ibm/ws/security/fat/common/expectations/ResponseFullExpectation.java

```diff
@@ -25,6 +25,10 @@ public ResponseFullExpectation(String testAction, String checkType, String searc
         super(testAction, Constants.RESPONSE_FULL, checkType, searchFor, failureMsg);
     }
 
+    public ResponseFullExpectation(String testAction, String searchLocation, String checkType, String searchKey, String searchFor, String failureMsg) {
+        super(testAction, searchLocation, checkType, searchKey, searchFor, failureMsg);
```

Comment:
I'm confused by this one - the point of it being a `ResponseFullExpectation` object is because the search location is the full response, and always will be the full response. An expectation that searches somewhere else would be a different type of expectation.

Reply:
it was in because of a method in JwtApiExpectations - both being removed with issue 10532

Reply:
After more digging (and stopping/starting my eclipse to see that the method in JwtApiExpectations was needed), I can't remove this - I need to create a JwtTimeClaimExpectation with these parms, that expectation extends JwtApiExpectation which in turn extends ResponseFullExpectation.  I can't access the contructor without this and the one in JwtApiExpectations being defined.  (I need a JwtTimeClaimExpectation so that I get the override validation method that does special time comparisons in the full response).  The constructors aren't inherited, so I need each method used in each of the classes.